Gemfile

```diff
@@ -27,6 +27,8 @@ gem 'gettext_i18n_rails', '~> 1.0'
 gem 'i18n', '~> 0.6.4'
 gem 'turbolinks', '~> 2.5'
 gem 'logging', '>= 1.8.0', '< 3.0.0'
+gem 'activerecord-session_store'
+gem 'rails-observers'
```

Comment:
i wonder if we should not move the only observer that we have back into the host class.